app/views/admins/rewards/_form.html.erb

```diff
@@ -4,6 +4,28 @@
   <%= form.text_field :description %>
   <%= form.number_field :price, min: 1, step: 1 %>
 
+  <input name="reward[category_ids][]" type="hidden" value>
```

Comment:
Bulma extention TagsImput is used here. 

Hidden input is there to send empty array when all categories are deleted (multiselect form helper in rails does this).